tensorflow/inference/docker/2.8/py3/Dockerfile.cpu

```diff
@@ -94,6 +94,7 @@ RUN ${PIP} install --no-cache-dir \
     "awscli<2" \
```

Comment:
On line [106](https://github.com/aws/deep-learning-containers/pull/2538/files#diff-d17c4581f9b288b81580b22413774fc2b8c706090ac4931f9c94a23550d41380L105), can we please change the version to`tensorflow-serving-api=="2.8.4"`? 

Thanks!!